### ISSUE TITLE
Crashes calling NSBundle(UINSBundleAdditions) loadNibNamed:owner:options:

### ISSUE THREAD

**Issue by wutschel** [xbmc/Official-Kodi-Remote-iOS#678]
There are crashes reported when calling `NSBundle(UINSBundleAdditions) loadNibNamed:owner:options:`. One example can be found here: [crashlog](https://www.dropbox.com/s/6kxzf31ob4ed5x8/DPTu-BcVYS6Z3QdBaZpAFZ.xccrashpoint.zip?dl=0). `loadNibNamed` is called in 7 locations in the code, 6x inside `cellForRowAtIndexPath` and 1x inside `initWithFrame`.

`initWithFrame` snipped looks like this:
```
...
if (self) {
    NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"VolumeSliderView" owner:self options:nil];
    self = nib[0];
...
```
`cellForRowAtIndexPath` snippet looks like this (first one is the location of one of the reported crashes):
```
...
UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"rightMenuCell"];
[[NSBundle mainBundle] loadNibNamed:@"rightCellView" owner:self options:NULL];
if (cell == nil) {
    cell = rightMenuCell;
   ...
```
or
```
...
if (cell == nil) {
    NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"jsonDataCell" owner:self options:nil];
    cell = nib[0];
    ....
```

@kambala-decapitator, can you already tell from the code snippet where there is wrong solution taken? The code is pretty old and was not really changed (except coding styles) since a while.

**Comment by kambala-decapitator** [comment]
first snippet looks wrong (nib owner is destroyed), other seem fine

**Comment by wutschel** [comment]
Rewriting the `UIView`'s `initWithFrame` snippet to directly load the `nib` would be ok?

```
- (id)initWithFrame:(CGRect)frame leftAnchor:(CGFloat)leftAnchor isSliderType:(BOOL)isSliderType {
    NSArray *nib = [[NSBundle mainBundle] loadNibNamed:@"VolumeSliderView" owner:self options:nil];
    self = nib[0];
    if (self) {
   ...
```

**Comment by wutschel** [comment]
Since I opened this issue there are a few more instances which show the crash is mainly caused when calling `cellForRowAtIndexPath` from an animation (to fade out and fade in the cells). Is this in any way problematic?

As the animation is not required I added a change to ease up the implementation and remove it. The changes now include the code snippet from last comment and the removal:
https://github.com/wutschel/Official-Kodi-Remote-iOS/commits/load_volume_nib
